guides/src/main/java/io/micronaut/guides/core/DefaultWebsiteGenerator.java

```diff
@@ -34,41 +34,44 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * Default implementation of the {@link WebsiteGenerator} interface.
+ * This class is responsible for generating a website from the specified input directory to the specified output directory.
+ */
 @Internal
 @Singleton
 class DefaultWebsiteGenerator implements WebsiteGenerator {
     private static final Logger LOG = LoggerFactory.getLogger(DefaultWebsiteGenerator.class);
     private static final String FILENAME_TEST_SH = "test.sh";
     private static final String FILENAME_NATIVE_TEST_SH = "native-test.sh";
     private static final String FILENAME_INDEX_HTML = "index.html";
+
     private final GuideParser guideParser;
     private final GuideProjectGenerator guideProjectGenerator;
     private final JsonFeedGenerator jsonFeedGenerator;
-    private final JsonFeedConfiguration jsonFeedConfiguration;
     private final RssFeedGenerator rssFeedGenerator;
-    private final RssFeedConfiguration rssFeedConfiguration;
     private final FilesTransferUtility filesTransferUtility;
     private final TestScriptGenerator testScriptGenerator;
     private final MacroSubstitution macroSubstitution;
     private final AsciidocConverter asciidocConverter;
     private final IndexGenerator indexGenerator;
     private final GuideMatrixGenerator guideMatrixGenerator;
     private final GuideProjectZipper guideProjectZipper;
+    private final ConfigurationsProvider configurationsProvider;
```

Comment:
@sdelamo this is because checkStyle was complaining about having 13 parameters (max is 12)